frontend/js/models/track.js

```diff
@@ -46,7 +46,7 @@ define([
              * @static
              */
             defaults: {
-                access: ACCESS.PRIVATE,
+                access: ACCESS.SHARED_WITH_ADMIN,
```

Comment:
We discussed that in the public meetings as we had usability issues in classrooms (in courses, the students tend to forget to share it with admins, so the work could frequently not be reviewed)